src/backend/main/py.main/caliopen_main/discussion/store/discussion_index.py

```diff
@@ -69,11 +69,19 @@ def list_discussions(self, limit=10, offset=0, min_pi=0, max_pi=0):
         ids, total = self.__search_ids(limit, offset, min_pi, max_pi)
         discussions = []
         for id, count in ids.items():
-            message = self._get_last_message(id, min_pi, max_pi)
+            message = self.get_last_message(id, min_pi, max_pi)
             discussions.append(message)
         # XXX total do not work completly, hack a bit
         return discussions, total + len(discussions)
 
-    def get_discussion(self, discussion_id, min_pi, max_pi):
-        """Get a specific discussion."""
-        return self._get_last_message(discussion_id, min_pi, max_pi)
+    def get_a_message_id(self, discussion_id, message_id):
+        """Search a message_id within a discussion"""
+
+        search = self._prepare_search(0, 100)
+        search.body = {"query": {"match_all":
+                                     {"discussion_id": discussion_id,
+                                      "message_id": message_id}}}
+        result = search.execute()
+        if not result.hits:
+            return False
```

Comment:
can't return a boolean or an object (document). return type have to be consistent (return None or {})